### PR TITLE
Support for Secondary RocksDB Instances

### PR DESCRIPTION
**Pull request by roeeiit1**

### CODE REVIEW

**Comment by roeeiit1** [comment]
Hi @theolivenbaum  
Wanted to know if there is anything else you need me to add in order to complete this pull request 🙂
Thanks in advance

**Comment by almogtavor** [comment]
A very anticipated feature

**Comment by theolivenbaum** [comment]
Hi everyone! PR looks good, didn't know about this feature before. Think only thing to fix is to move the dependencies for testing to the test project and remove it from the library project

File: csharp/RocksDbSharp.csproj [hunk 1]
```diff
@@ -26,6 +26,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.8" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.8" />
```
Comment (theolivenbaum) [inline csharp/RocksDbSharp.csproj:31]: These dependencies should only be in the test project

**Comment by roeeiit1** [comment]
Hey, sure! 
Is there an existing test project or should I create one? 
Thank you for the quick response! :D

**Comment by theolivenbaum** [comment]
Apologies! I thought there was one - but was confusing with another one of our repos. If you could move it then to a separate project, that would be better, so we don't introduce the test SDKs as dependencies of the library! Thanks!

**Comment by almogtavor** [comment]
Why would let the tests be in the same repo? This doesn't mean they have to be included in the SDK

**Comment by roeeiit1** [comment]
No he means to leave them in the repo just put them in another project as the project is closed as the package in the end :)

**Comment by roeeiit1** [comment]
@theolivenbaum Done! :)
btw i also added support for compiling the dll for dotnet core 3.1 if thats ok with you as i dont see anything that makes it conflict there and i am still using dotnet core 3.1 in my app 😅
can remove that if needed or open a sperate pr for it

**Comment by almogtavor** [comment]
Looks great to me

File: csharp/RocksDbSharp.csproj (hunk 1 above)
Comment (roeeiit1) [reply csharp/RocksDbSharp.csproj]: Done :)

**Comment by theolivenbaum** [comment]
Merged!

**Pull request merged**